src/transformers/modeling_tf_utils.py

```diff
@@ -2475,12 +2476,95 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
 
         return model
 
+    def push_to_hub(
+        self,
+        repo_id: str,
+        use_temp_dir: Optional[bool] = None,
+        commit_message: Optional[str] = None,
+        private: Optional[bool] = None,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        max_shard_size: Optional[Union[int, str]] = "10GB",
+        **model_card_kwargs
+    ) -> str:
+        """
+        Upload the model files to the 🤗 Model Hub while synchronizing a local clone of the repo in `repo_path_or_name`.
 
```

Comment:
Small detail: does it make sense to refer to `repo_id` instead of `repo_path_or_name` since the later seems to be deprecated ?

Reply:
Sorry, I just realised it is a merged PR anyway 😬

Reply:
I can still address ;-)